src/lib/AnnotationControls.ts

```diff
@@ -65,6 +64,8 @@ export default class AnnotationControls {
 
     private hasInit = false;
 
+    private isModeTemp = false;
```

Comment:
Should this be used to control whether the button is marked as "active' or not? What does this actually do?

Reply:
I changed the name to `isTemporary` and it controls whether the current active control is temporary or not. If it's temporary, when we receive "annotations creation success" or "staged cancel", we reset the active control to none.